moveit_core/robot_state/test/robot_state_benchmark.cpp

```diff
@@ -32,132 +32,338 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  *********************************************************************/
 
-/* Author: Robert Haschke */
+/* Author: Mario Prats, Robert Haschke */
```

Comment:
Maybe put yourself 2nd to make clearer who the original author is.

Reply:
sg